media/css/sandstone/buttons.less

```diff
@@ -333,6 +333,15 @@ ul.download-list {
     display: block !important;
 }
 
+// Add another rule for old browsers where :not isn't supported. If this is
+// mixed with the rule above, the entire rule will be invalid on those legacy
+// environments and some tests will fail.
+.osx:not(.pre-mavericks) .download-button .os_osx,
+.osx.pre-mavericks .download-button:not(.mavericks-required) .os_osx {
+    display: block !important;
+}
```

Comment:
This looks to be fine, as the only OS/browser combination to have trouble would be Firefox 3 on OS X 10.6 - 10.8, which is probably an *extremely* rare edge case. However, I believe those users would not see a download button at all. Is that right?

(I asked in #growth and #www about the number of Fx 3 users still out there, but haven't heard back.)

What would you think about displaying `.osx .download-button .os_osx` by default, then using the `:not` selector to hide them (where appropriate) from modern browsers?